SPECS/hwdata/hwdata.spec

```diff
@@ -1,20 +1,29 @@
 Summary:        Hardware identification and configuration data
 Name:           hwdata
-Version:        0.356
+Version:        0.378
 Release:        1%{?dist}
-License:        GPLv2+ OR XFree86 1.0
+License:        GPL-2.0-or-later
```

Comment:
This should not have changed based on the License I see on github. Is there a reason it was changed?
https://github.com/vcrhonek/hwdata/blob/master/LICENSE . 
```
This data is licenced under 2 different licenses

1) GNU General Public License, version 2 or later
2) XFree86 1.0 license

```

Reply:
Thanks for pointing it out! The change is due to a misunderstanding, reverted it to the correct one.